utils/run_test.sh

```diff
@@ -153,11 +188,15 @@ if [ -d "/mnt/daos" ]; then
     run_test src/common/tests/btree.sh dyn -s 20000
     run_test src/common/tests/btree.sh dyn perf -s 20000
     run_test src/common/tests/btree.sh dyn perf ukey -s 20000
+
+    COMPONENT="UT_vos"
     run_test src/vos/tests/evt_ctl.sh
     run_test src/vos/tests/evt_ctl.sh pmem
     unset USE_VALGRIND
     unset VALGRIND_SUPP
 
+    mv ${DAOS_BASE}/test_results/xml/*.xml ${DAOS_BASE}/test_results
```

Comment:
(lint) Double quote to prevent globbing and word splitting. [SC2086]
(lint) Double quote to prevent globbing and word splitting. [SC2086]